src/test/java/com/iexec/core/logs/TaskLogsServiceTests.java

```diff
@@ -66,6 +62,32 @@ void shouldAddComputeLogs() {
         assertThat(capturedEvent.getComputeLogsList().get(0).getWalletAddress()).isEqualTo(WORKER_ADDRESS);
     }
 
+    @Test
+    void shouldNotAddComputeLogsSinceNull() {
+        taskLogsService.addComputeLogs(CHAIN_TASK_ID, null);
+        verifyNoInteractions(taskLogsRepository);
+    }
+
+    @Test
+    void shouldNotAddComputeLogsSinceLogsAlreadyKnown() {
+        final ComputeLogs computeLogs = new ComputeLogs(WORKER_ADDRESS, STDOUT, STDERR);
+        final TaskLogs taskLogs = TaskLogs.builder()
+                .chainTaskId(CHAIN_TASK_ID)
+                .computeLogsList(Collections.singletonList(computeLogs))
+                .build();
+
+        when(taskLogsService.getTaskLogs(CHAIN_TASK_ID))
+                .thenReturn(Optional.of(taskLogs));
+
+        ArgumentCaptor<TaskLogs> argumentCaptor = ArgumentCaptor.forClass(TaskLogs.class);
```

Comment:
This does not seem to be used.
Do we need it ?

Reply:
You're right, we don't :)